contributingGuides/PROPOSAL_TEMPLATE.md

```diff
@@ -7,6 +7,9 @@
 ### What changes do you think we should make in order to solve the problem?
 <!-- DO NOT POST CODE DIFFS -->
 
+### What specific scenarios should we cover in unit tests to prevent reintroducing this issue in the future?
+<!-- Clearly describe the different test cases you recommend adding or updating. Explain how they will ensure the problem is fully covered and that any future changes do not cause a regression. Consider edge cases, input variations, and typical user interactions that could trigger this issue. To get guidance on how to write unit tests, refer to the README.md in the tests folder. -->
```

Comment:
We say 'To get guidance on how to write unit tests' here, but I believe we want all changes to be under test which includes UI/component testing. Should we update this to link to both unit and component testing folders?

Reply:
Perhaps we are including component tests under 'unit', which is fine but to a contributor they might think: 'oh well I modified this components state  when another component is modified, therefore I don't need to add a unit test'

Reply:
Updated the proposal template 🙌